weave-js/src/panel/WeaveExpression/styles.ts

```diff
@@ -34,6 +34,10 @@ export const WeaveEditable = styled(Editable)<{$truncate?: boolean}>`
   font-family: Inconsolata;
   cursor: text;
 
+  &:focus {
+    outline: 0;
+  }
```

Comment:
When we upgrade slate, we get the blue outline. To keep things consistent with existing functionality, I removed it. However, this seems like a good thing to keep


https://github.com/wandb/weave/assets/103448723/d433ee03-2dfb-4dc0-8bed-8dae66310ef1